### ISSUE TITLE
Can't create new line in file.

### ISSUE THREAD

**Issue by ehackett00** [microsoft/vscode#123758]
We have written the needed data into your clipboard because it was too large to send. Please paste.
Issue Type: <b>Bug</b>

1. Open Java Script  file in VS Code.
2.   Place cursor at end of existing line or at a blank line.
3.  Hit return on Mac.
==> No new line created.
Tried new file and other text files; no luck.

VS Code version: Code 1.56.0 (Universal) (cfa2e218100323074ac1948c885448fdf4de2a7f, 2021-05-04T22:06:21.189Z)
OS version: Darwin x64 20.4.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.46GB free)|
|Process Argv|--crash-reporter-id dd9896a9-49e2-4a58-b174-1bb0739bccfa|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.1
cocos-debug|Chu|0.0.1
zsh-completion-script|dua|1.0.0
vscode-npm-script|eg2|0.3.21
vscode-firefox-debug|fir|2.9.4
code-runner|for|0.11.3
cloudcode|goo|1.11.1
cocotools|jas|0.0.7
file-downloader|min|1.0.10
vscode-docker|ms-|1.12.1
vscode-kubernetes-tools|ms-|1.3.2
remote-containers|ms-|0.177.2
cpptools|ms-|1.3.1
debugger-for-chrome|msj|4.12.12
vscode-janus-debug|otr|1.3.1
vscode-yaml|red|0.18.0
zsh-debug|rog|0.1.3
vim|vsc|1.20.2
quokka-vscode|Wal|1.0.366

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30291414
vspre833:30267464
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30286553
vscorehov:30301222

```

</details>

**Comment by ehackett00** [comment]
Hello,

I’’m sorry,  I did not see a solution in other issues.  It might be because I am blind and my computer can’t read it.  Can you tell me how these issues were fixed?

Ed

**Comment by niccolomineo** [comment]
I have exactly the same problem since 1.56.0 and it's very annoying.

Steps to reproduce the problem:

- Rename a file from the sidebar tree and hit ENTER at the end
- Try to hit enter, you will receive this error:

<img width="468" alt="Schermata 2021-05-13 alle 11 23 59" src="https://user-images.githubusercontent.com/5409469/118106404-de151500-b3dd-11eb-947c-292a04dfe20e.png">

In addition, I received another error yesterday (mentioning an invalid integer operation or equality) but I am not unable to report the steps to reproduce it, at the moment.

**Comment by dostuffthatmatters** [comment]
I have the exact same error as @niccolomineo 

I have noticed that this behavior occurs every time I add a new terminal tab (with the new tab-UI).

**Comment by niccolomineo** [comment]
I updated my report above with more consistent steps to reproduce the issue (and, yes, it revolves around the new tab UI, like @dostuffthatmatters said).

**Comment by gjsjohnmurray** [comment]
/duplicate of #123040 and will be fixed when 1.56.2 arrives shortly.
